css/style.scss

```diff
@@ -621,6 +621,20 @@ input.input-inline {
 		font-size: 8pt !important;
 		display: none;
 		align-self: flex-end;
+		min-width: 120px;
```

Comment:
IMO this is not needed, because it doesn't look bad if `.save-indicator` and `.unsave-indicator` do not have identical with. Also, if identical with is what you were aiming at, min-width will not solve the problem - what if one of the strings is longer than120px in another language?

Reply:
You're right, it also looks fine without.